crates/core/src/parameters.rs

```diff
@@ -46,6 +46,9 @@ pub struct Parameters {
     pub implicit_vp_code_hash: Option<Hash>,
     /// Expected number of epochs per year (read only)
     pub epochs_per_year: u64,
+    /// The multiplier for masp epochs (it requires this amount of epochs to
+    /// transition to the next masp epoch)
+    pub masp_epoch_multiplier: u64,
```

Comment:
```suggestion
    pub masp_epoch_multiplier: NonZeroU64,
```

Reply:
I like this one but we have also other parameters that could benefit from this. Should this become a wider refactor?

Reply:
yeah, replacing primitive integer types with their nonzero equivalents where prudent

Reply:
Definitely helpful, because if max_proposal_bytes_key is set to 0, it will cause a consensus failure, crash the nodes and halt the chain.

Also, setting max_block_gas to 0, or at least lower than the minimum required for a transaction, will make the chain non-interactable, but the protocol people told me that is acceptable risk.